internal/livetiming/livetiming.go

```diff
@@ -37,3 +44,18 @@ func Init() error {
 
 	return err
 }
+
+func Info(year, round int64) (*Session, error) {
+	var (
+		sess Session
```

Comment:
generally speaking we favor initializing variables with the `:=` shorthand as they're introduced, though if there's a good reason for declaring at the top it might be a fun discussion for the guild.

Reply:
This was mostly due to wanting to keep the 'exit early' philosophy alongside the `if err := stmt(); err != nil {` structure. If `sess` was not declared ahead of time, it would only be created on L56, and only in the scope of the `if` block. 

It can be avoided by just separating the `val, ok := map[key]` check from the `if !ok` check. I can do that in the future, I just tend to prefer the pre-init and `if err := ...` format like this for map validation since imo it makes it neater. But its really not a big deal, so if thats the style that'll be easy enough to follow!